_opensearch/bucket-agg.md

```diff
@@ -709,7 +709,19 @@ GET opensearch_dashboards_sample_data_logs/_search
  }
 }
 ```
+When you add a document to an index with `ip_range` mappings set, if there are any malformed fields in the document, the entire document will get rejected. You can use the `ignore_malformed` parameter set to `true` to ignore any malformed IP ranges. This allows you to add the document and ignore the malformed fields.
 
+```json
+...
+"mappings": {
+    "properties": {
+        "ips": {
+            "type": "ip_range",
+            "ignore_malformed": true
+        }
+    }
+}
```

Comment:
This is an unfair nitpick, but to stay consistent with the rest of the site, we want to indent code blocks with spaces instead of indents. For each level of indentation, I use two spaces.

Reply:
thanks